### ISSUE TITLE
New MMR script v1.1 shows an MMR difference at the start of the game that shouldn't be there

### ISSUE THREAD

**Issue by XenSide** [AveYo/D-OPTIMIZER#3]
When you start Dota and get to the main menu for the first time it shows the bar like you've just lost with absurd amount of MMRs

I.E:

09/19/19 20:08:33  Behavior: NORMAL  Core: 4348  +0  Support: 4275  -423459  

This is taken from the console but it's actually displaying the top bar aswell.

Also, I would like to add that I would love the option to have something to export this values after all the games to a csv or JSON file.

What I've done to try and do that is send the command 
SendToServerConsole('clear;cl_class;condump;')

And that kinda works but what it does is create a new and different dump file every single time this is called so it's not really perfect.

**Comment by AveYo** [comment]
I have already some ideas on how to improve it, and might even provide a native logging to file :)

**Comment by XenSide** [comment]
Big fan of all your work and this has yet again peaked my interest, like every time you make something!

Hope you can work on the logging soon, it's something I'm really looking forward to! (And according to the reddit post not only me!)

**Comment by AveYo** [comment]
Yep, got logging to file working, it's just a matter of making the filtering more reliable and then will update it!

**Comment by XenSide** [comment]
You're actually incredible, In my last answer I forgot to add that I kinda managed to log to file using clear;cl_class;condump but that created a new file everytime and I guess you already figured it out anyways.

The problem with the first bar appearing is related to getting the old support mmr, it always counts it as a 6 digit number (close to 500000) when the core mmr is actually always good.   
Also, it only happens at the start of the game, if you actually check cl_class right after or try to type disconnect in console without even checking it won't show up and the cl_class mmr will be right.

**Comment by xaow1q** [comment]
Idk how, cauz i'm not a programmer or smth like this, but please fix that, after 2 wins or more, 2 loses or more your script show sumarized mmr , not won/lost mmr for a 1 game

**Comment by churchymayer** [comment]
I have the same issue where i'm "losing" an absurd amount of support MMR every time i start up the game. Reading through this thread I don't see a fix for this. Is this something we can expect to be fixed soon or is there something I can do on my side?
